rubyparser.h

```diff
@@ -804,6 +804,8 @@ typedef struct RNode_BLOCK_PASS {
     struct RNode *nd_head;
     struct RNode *nd_body;
     unsigned int forwarding: 1;
+    rb_code_location_t name_loc;
```

Comment:
IIUC, `name_loc` will be handled by node on `NODE_BLOCK_PASS#nd_body` instead of `NODE_BLOCK_PASS`.

Reply:
Indeed, I fixed it. Thank you.

```
❯ bin/parse -e "foo(&a)"
@ ProgramNode (location: (1,0)-(1,6))
├── flags: ∅
├── locals: []
└── statements:
    @ StatementsNode (location: (1,0)-(1,6))
    ├── flags: ∅
    └── body: (length: 1)
        └── @ CallNode (location: (1,0)-(1,6))
            ├── flags: newline, ignore_visibility
            ├── receiver: ∅
            ├── call_operator_loc: ∅
            ├── name: :foo
            ├── message_loc: (1,0)-(1,3) = "foo"
            ├── opening_loc: (1,3)-(1,4) = "("
            ├── arguments: ∅
            ├── closing_loc: (1,6)-(1,7) = ")"
            └── block:
                @ BlockArgumentNode (location: (1,4)-(1,6))
                ├── flags: ∅
                ├── expression:
                │   @ CallNode (location: (1,5)-(1,6))
                │   ├── flags: variable_call, ignore_visibility
                │   ├── receiver: ∅
                │   ├── call_operator_loc: ∅
                │   ├── name: :a
                │   ├── message_loc: (1,5)-(1,6) = "a"
                │   ├── opening_loc: ∅
                │   ├── arguments: ∅
                │   ├── closing_loc: ∅
                │   └── block: ∅
                └── operator_loc: (1,4)-(1,5) = "&"
```